### DISCUSSION TITLE
LYB printer

### DISCUSSION THREAD

**Discussion by pkmuthu1983** [CESNET/libyang#2193] [Q&A]
It seems like we can only print top-level nodes in the LYD_LYB format. If other nodes are passed, I get the error:
 "LYB printer supports only printing top-level nodes."

Will this be supported in upcoming libyang versions ?

**Comment by michalvasko** [comment]
No, there is no plan to, currently, as we have no use-case and it would require some changes in the LYB format making it incompatible with the current one.
